src/client/common.py

```diff
@@ -384,7 +384,7 @@ def curlify(service, req_str, gnucli=False, execute=False, prettyprint=False):
             pp.pprint(_remove_item_keys(do_request(verb, request_string)))
             return
 
-        print _remove_item_keys(do_request(verb, request_string), gnucli)
```

Comment:
Looks good. Just a question as I don't know much about the 'gnucli' argument part: will this change in line 387 affect any other CLIs?